docs/src/modules/components/AppNavDrawer.js

```diff
@@ -653,9 +653,13 @@ export default function AppNavDrawer(props) {
               ])}
             />
           )}
-          {asPathWithoutLang.startsWith('/x/advanced-components') && (
-            <ProductIdentifier name="Advanced components" metadata="MUI X" />
-          )}
+          {
+            // TODO: remove first condition when https://github.com/mui/mui-x/pull/4692 is released
+            (asPathWithoutLang.startsWith('/x/advanced-components') ||
+              asPathWithoutLang.startsWith('/x/getting-started')) && (
+              <ProductIdentifier name="Advanced components" metadata="MUI X" />
+            )
+          }
```

Comment:
I wonder. Why are we not doing this

```suggestion?
          {
            (asPathWithoutLang.startsWith('/x/') && (
              <ProductIdentifier name="Advanced components" metadata="MUI X" />
            )
          }
```

for all the X pages. Why do we treat different side nav differently? It sounds simpler & clearer.

Reply:
Hmm, I agree. If we are not going to put Data Grid and Date Pickers to the menu (as we have Material UI, MUI Base and MUI System under MUI Core) - then it makes more sense to always show "Advanced components" there.

Reply:
Plus, the version, we don't version the Data grid differently from the Date picker, it's the same version strategy. Same goes when it comes to MUI Base vs. Material UI vs. Joy UI. It's a single version (with some package in alpha)

Reply:
> we don't version the Data grid differently from the Date picker

We do - currently data grid is at 5.10.0 and pickers - at v5.0.0-alpha.2.
Even when releasing pickers as stable and synchronizing the versions - data grid should have v4 in the versions list (as it does now), while date pickers shouldn't (there was no v4 of pickers package).
I think that's a solid argument to keep product identifier menu as it is now.

Moreover, few times I was disappointed that I can't switch from Data Grid docs to Date Pickers docs using product identifier. Maybe we should consider listing the packages there?